src/libmongoc/src/mongoc/mongoc-async-cmd.c

```diff
@@ -484,3 +506,39 @@ _mongoc_async_cmd_phase_recv_rpc (mongoc_async_cmd_t *acmd)
 
    return MONGOC_ASYNC_CMD_IN_PROGRESS;
 }
+/*
```

Comment:
I think this copyright notice and included headers are not intended to be included.